db/db_test2.cc

```diff
@@ -3355,16 +3390,18 @@ TEST_F(DBTest2, MemtableOnlyIterator) {
 
   // Memtable-only iterator (read_tier=kMemtableTier); data not flushed yet.
   it = db_->NewIterator(ropt, handles_[1]);
+  ASSERT_OK(it->status());
```

Comment:
Not sure this line is necessary, but can leave it if you want.

Reply:
You are right. I have removed it.